share/spice/twitch_streams/twitch_streams.js

```diff
@@ -0,0 +1,69 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_twitch_streams = function(api_result){
+
+        if (api_result.error) {
```

Comment:
Done and done! =)
Followed the tutorial, maybe a note should be added to the doc? https://duck.co/duckduckhack/spice_basic_tutorial